### PR TITLE
plugin docs

### PR DESCRIPTION
**Pull request by odra**
Adding usage docs.

### CODE REVIEW

File: docs/README.adoc [hunk 1]
```diff
@@ -0,0 +1,81 @@
+== Aerogear Kryptowire Jenkins Plugin
+
+This Jenkins plugins integrates with the Kryptowire plugin by adding the following features:
```
Comment (mikenairn) [inline docs/README.adoc]: Do you mean "integrates with the Kryptowire service" here?

Comment (mikenairn) [reply docs/README.adoc]: or maybe platform, just not sure you meant plugin.

File: docs/README.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+=== Usage
+
+=== Kryptowire Global Configuration
+
+You can set your Kryptowire global configuration in Jenkins configuration page (usually by acessing the `/configure` route).
+
+You can quickly find the Kryptowire section from the top menu:
+
+image::images/image-1.png[]
```
Comment (mikenairn) [inline docs/README.adoc:24]: Are these image paths correct?

Comment (odra) [reply docs/README.adoc:24]: Yeah, just fixed it by putting the images in the right place

**Review by mikenairn — APPROVED** [review]
Looks good :+1:

**Pull request merged**